pkg/controller/eventing/inmemory/clusterchannelprovisioner/reconcile.go

```diff
@@ -62,87 +62,87 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
-	err := r.client.Get(ctx, request.NamespacedName, cp)
+	ccp := &eventingv1alpha1.ClusterChannelProvisioner{}
+	err := r.client.Get(ctx, request.NamespacedName, ccp)
 
-	// The ClusterProvisioner may have been deleted since it was added to the workqueue. If so,
+	// The ClusterChannelProvisioner may have been deleted since it was added to the workqueue. If so,
 	// there is nothing to be done.
 	if errors.IsNotFound(err) {
-		logger.Info("Could not find ClusterProvisioner", zap.Error(err))
+		logger.Info("Could not find ClusterChannelProvisioner", zap.Error(err))
 		return reconcile.Result{}, nil
 	}
 
 	// Any other error should be retried in another reconciliation.
 	if err != nil {
-		logger.Error("Unable to Get ClusterProvisioner", zap.Error(err))
+		logger.Error("Unable to Get ClusterChannelProvisioner", zap.Error(err))
 		return reconcile.Result{}, err
 	}
 
-	// Does this Controller control this ClusterProvisioner?
-	if !shouldReconcile(cp.Namespace, cp.Name, cp.Spec.Reconciles.Kind) {
-		logger.Info("Not reconciling ClusterProvisioner, it is not controlled by this Controller", zap.String("APIVersion", cp.APIVersion), zap.String("Kind", cp.Kind), zap.String("Namespace", cp.Namespace), zap.String("name", cp.Name))
+	// Does this Controller control this ClusterChannelProvisioner?
+	if !shouldReconcile(ccp.Namespace, ccp.Name) {
+		logger.Info("Not reconciling ClusterChannelProvisioner, it is not controlled by this Controller", zap.String("APIVersion", ccp.APIVersion), zap.String("Kind", ccp.Kind), zap.String("Namespace", ccp.Namespace), zap.String("name", ccp.Name))
 		return reconcile.Result{}, nil
 	}
-	logger.Info("Reconciling ClusterProvisioner.")
+	logger.Info("Reconciling ClusterChannelProvisioner.")
 
 	// Modify a copy of this object, rather than the original.
-	cp = cp.DeepCopy()
+	ccp = ccp.DeepCopy()
 
-	err = r.reconcile(ctx, cp)
+	err = r.reconcile(ctx, ccp)
 	if err != nil {
-		logger.Info("Error reconciling ClusterProvisioner", zap.Error(err))
+		logger.Info("Error reconciling ClusterChannelProvisioner", zap.Error(err))
 		// Note that we do not return the error here, because we want to update the Status
 		// regardless of the error.
 	}
 
-	if updateStatusErr := r.updateClusterProvisionerStatus(ctx, cp); updateStatusErr != nil {
-		logger.Info("Error updating ClusterProvisioner Status", zap.Error(updateStatusErr))
+	if updateStatusErr := r.updateClusterChannelProvisionerStatus(ctx, ccp); updateStatusErr != nil {
+		logger.Info("Error updating ClusterChannelProvisioner Status", zap.Error(updateStatusErr))
 		return reconcile.Result{}, updateStatusErr
 	}
 
 	return reconcile.Result{}, err
 }
 
 // IsControlled determines if the in-memory Channel Controller should control (and therefore
-// reconcile) a given object, based on that object's ClusterProvisioner reference. kind is the kind
+// reconcile) a given object, based on that object's ClusterChannelProvisioner reference. kind is the kind
```

Comment:
the last sentence: kind is the kind... is no longer relevant.

Reply:
removed :)